app/locker/data/dashboards/LrsDashboard.php

```diff
@@ -52,19 +52,40 @@ public function statementCount(){
    *
    **/
   public function actorCount(){
-
-    $count = $this->db->statements->aggregate(
+    $count_array = ['mbox' => '', 'openid' => '', 'mbox_sha1sum' => '', 'account' => ''];
+    
+    $count_array['mbox'] = $this->db->statements->aggregate(
               array('$match' => $this->getMatch( $this->lrs )),
```

Comment:
If possible, please convert the `array(...)` to `[...]` syntax, to maintain consistency and improve readability.